nodejs/v3/nextjs-appdir/commands/run

```diff
@@ -0,0 +1,29 @@
+#!/bin/bash
+
+set -eu
+
+echo "Installing compatible NPM version"
+npm install -g npm@latest
+
+# echo "Install, link and build integration"
+# (
+# cd /integration
+# git checkout main
+# touch ~/.bashrc # hack to make the mono installer happy
+# script/setup
+# mono bootstrap
+# mono clean
+# mono run -- npm run install
+# mono build
+# )
```

Comment:
Thank you! Yes, I will definitely leave a comment here. And I will also add a note to a README somewhere, saying that, unlike the other test setups, these two don't use the copy of the integration that's cloned by `integrations:clone`, instead using the latest released version.

Note that the instructions in the terrible nonsense of a process are only necessary _right now_, because there is no published Node.js release that has the agent with the Next.js extractor. Once that's merged and released, none of that will be necessary, and `rake app=nodejs/v3/nextjs-appdir app:up` will be all that's needed.

Reply:
Done: https://github.com/appsignal/test-setups/pull/109/commits/9de04cdba4096553c4bfc611bfa8f6c639d7a7b7